### PR TITLE
Python release 9.7.1 -- MERGE March 11 PM

### PR DESCRIPTION
**Pull request by hmstepanek**
## Give us some context

* Add support notes for Python release 9.7.1.
* Fix incorrect default setting.

### CODE REVIEW

File: src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-90701.mdx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+subject: Python agent
+releaseDate: '2024-02-22'
```
Comment (hmstepanek) [inline src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-90701.mdx:3]:
```suggestion
releaseDate: '2024-03-07'
```

File: src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-90701.mdx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+This release of the Python agent fixes a bug in the NewRelicContextFormatter where attributes were dropped from logs.
+
+Install the agent using `easy_install/pip/distribute` via the [Python Package Index](https://pypi.python.org/pypi/newrelic) or download it directly from the [New Relic download site](https://download.newrelic.com/python_agent/release).
+
+
+## Bug fixes
+
+* **Fix log attribute dropping in NewRelicContextFormatter**
+    *  In v9.5.0, a bug in the NewRelicContextFormatter was introduced where "message" was added to the default set of keys to exclude. This caused extra attributes not in the default list to be dropped. This bug has been fixed.
```
Comment (hmstepanek) [inline src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-90701.mdx:21]:
```suggestion
    *  In v9.5.0, a bug in the `NewRelicContextFormatter` was introduced where `message` was added to the default set of keys to exclude. This caused extra attributes not in the default list to be dropped. This bug has been fixed. 
```

File: src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-90701.mdx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+## Notes
+
+This release of the Python agent fixes a bug in the NewRelicContextFormatter where attributes were dropped from logs.
+
+Install the agent using `easy_install/pip/distribute` via the [Python Package Index](https://pypi.python.org/pypi/newrelic) or download it directly from the [New Relic download site](https://download.newrelic.com/python_agent/release).
+
+
+## Bug fixes
+
+* **Fix log attribute dropping in NewRelicContextFormatter**
```
Comment (hmstepanek) [inline src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-90701.mdx:20]:
```suggestion
* **Fix log attribute dropping in `NewRelicContextFormatter`**
```

File: src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-90701.mdx [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+version: 9.7.1
+downloadLink: 'https://pypi.python.org/pypi/newrelic'
+features: []
+bugs: ['Fix log attribute dropping in NewRelicContextFormatter']
+security: []
+---
+
+## Notes
+
+This release of the Python agent fixes a bug in the NewRelicContextFormatter where attributes were dropped from logs.
```
Comment (hmstepanek) [inline src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-90701.mdx:13]:
```suggestion
This release of the Python agent fixes a bug in the `NewRelicContextFormatter` where attributes were dropped from logs.
```

**Comment by rhetoric101** [comment]
Temporarily closing to see if we can trigger a new, successful build.

**Review by rhetoric101 — APPROVED** [review]
@hmstepanek Thanks for adding these release notes!

**Pull request merged**